### PR TITLE
Create cinemasgaumontpathe.com.xml

### PR DESCRIPTION
**Pull request by nicolasleger**

### CODE REVIEW

**Comment by Bisaloo** [comment]
same issue as #9043.

**Comment by nicolasleger** [comment]
@Bisaloo Thanks for your help, I fixed it and regroup my commits in one.

File: src/chrome/content/rules/cinemasgaumontpathe.com.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<ruleset name="cinemasgaumontpathe.com">
+        <target host="cinemasgaumontpathe.com" />
+        <target host="www.cinemasgaumontpathe.com" />
+        <target host="m.cinemasgaumontpathe.com" />
+        <target host="pro.cinemasgaumontpathe.com" />
+        <target host="4dx.cinemasgaumontpathe.com" />
+        <target host="www.4dx.cinemasgaumontpathe.com" />
+        <target host="m-preprod.cinemasgaumontpathe.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/cinemasgaumontpathe.com.xml]: See also:
intranet.cinemasgaumontpathe.com
webadmin.cinemasgaumontpathe.com

File: src/chrome/content/rules/cinemasgaumontpathe.com.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+        <rule from="^http://cinemasgaumontpathe\.com/"
+                to="https://www.cinemasgaumontpathe.com/" />
+        <rule from="^http://4dx\.cinemasgaumontpathe\.com/"
+                to="https://www.4dx.cinemasgaumontpathe.com/" />
```
Comment (J0WI) [inline src/chrome/content/rules/cinemasgaumontpathe.com.xml]: Please document why those redirects are needed.

**Comment by J0WI** [comment]
@nicolasleger are you still interested to work on this?

**Comment by nicolasleger** [comment]
@J0WI sorry I don't have the time yet to re-enter in this, feel free to edit it.

**Comment by jeremyn** [comment]
@nicolasleger Are you still interested in working on this pull request? If not, please close this, or we will close this ourselves if we don't hear from you soon. If you want someone else to work on these domains, you can add them to issue https://github.com/EFForg/https-everywhere/issues/3069.

**Comment by J0WI** [comment]
@jeremyn I just edited the rule myself.

**Comment by jeremyn** [comment]
Delete entirely:
- [ ] https://m-preprod.cinemasgaumontpathe.com (gets a 403 with both HTTPS and plain HTTP)
- [ ] https://webadmin.cinemasgaumontpathe.com (gets a 403 with both HTTPS and plain HTTP)

Mark as `Time out`:
- [ ] https://intranet.cinemasgaumontpathe.com and remove the `target` (or delete this entirely because it is internal)

Add `target`:
- [ ] https://s.cinemasgaumontpathe.com

Also:
- [ ] Sort the `target`s (the `4dx` subdomains should go directly below `www`)
- [ ] Mark `^` as `Secure connection failed` in the top comment and remove the comment above the `rule`
- [ ] Change the `ruleset` `name` to `CinemasGaumontPathe.com`
- [ ] Change the filename to `CinemasGaumontPathe.com.xml`

**Comment by J0WI** [comment]
Done.

**Comment by jeremyn** [comment]
Thanks @J0WI , this is ready to go.

@nicolasleger Is it okay if I squash-and-merge all of these changes, including @J0WI 's, under your name in one commit?

**Comment by nicolasleger** [comment]
Yes all okay, sorry I didn't have the time to get back on those.

**Comment by jeremyn** [comment]
Okay thanks @nicolasleger and @J0WI , merged.

**Pull request merged**